features/keychain/module/crypto/seed-id.js

```diff
@@ -1,3 +1,5 @@
 import { fromMasterSeed } from '@exodus/bip32'
 
 export const getSeedId = (seed) => fromMasterSeed(seed).identifier.toString('hex')
+
+export const getManySeedIds = (seeds) => new Set(seeds.map(getSeedId))
```

Comment:
nit: I think we should either not make it a `Set` here or rename the function `getManyUniqueSeedIds`

Reply:
I ended up adding a new `getUniqueSeedIds` function here, but made it return an array to simplify things when handling the output